content/project-news/reactos-in-2020.md

```diff
@@ -0,0 +1,72 @@
+---
+title:       "ReactOS in 2020"
+author:      "Extravert34"
+date:        2020-12-28
+---
+
+Despite all the turbulence, we've got quite a rich year for ReactOS.
+Many bugs and instabilities were resolved, many more have been introduced. This year we hired two kernel developers full-time, this happens for the first time on the project history.
```

Comment:
Please start a new sentence on a new line (without an empty line between them).
This makes it simpler to view a diff, and still renders as one paragraph.